lib/graphql/query/serial_execution/field_resolution.rb

```diff
@@ -35,6 +35,7 @@ def result
         def get_finished_value(raw_value)
           if raw_value.is_a?(GraphQL::ExecutionError)
             raw_value.ast_node = irep_node.ast_node
+            raw_value.path = irep_node.path
```

Comment:
@rmosolgo I'm a little stumped on how to get the current collection index context here. Just want to double check its not available somehow that I'm just missing. Otherwise I can try to hack some extra state into the execution context.